help/en/releasenotes/current-draft-warnings.shtml

```diff
@@ -1,4 +1,10 @@
 
     <!-- contains new warnings for the release under development -->
 
-    <li>None yet</li>
+    <li>JMRI's scripting support now defaults to the UTF-8 character encoding for all
+        script files on all platforms. This may require that some scripts be re-saved
+        using UTF-8.</li>
+    <li><strong>Windows Only</strong> The JMRI installer removes the existing
+        <em>lib</em> folder from a JMRI installation when upgrading JMRI. The
```

Comment:
Maybe add something along the lines of `This means that any files you have added to the lib folder will be removed`.